planning/behavior_velocity_template_module/src/manager.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Tier IV, Inc., Leo Drive Teknoloji A.Ş.
```

Comment:
[nits] Fix copyright.

```suggestion
// Copyright 2023 TIER IV, Inc.
```

Reply:
Thank you, I missed that! 
It's fixed with last commit.